### PR TITLE
[SYCLomatic] Use getDpctNamespace() instead of hard code

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by wenhuiNi — APPROVED** [review]

**Review by tomflinda — APPROVED** [review]
LGTM

File: clang/lib/DPCT/APINamesCUB.inc [hunk 1]
```diff
@@ -891,360 +891,382 @@ CONDITIONAL_FACTORY_ENTRY(
...
-                       "cub::DeviceRadixSort::SortKeys",
-                       CALL("dpct::sort_keys",
-                             CALL("oneapi::dpl::execution::device_policy",
-                                  QUEUESTR),
-                             ARG(2), ARG(3), ARG(4), LITERAL("false"), ARG(5)))),
-                OTHERWISE(
-                     CALL_FACTORY_ENTRY(
-                       "cub::DeviceRadixSort::SortKeys",
-                       CALL("dpct::sort_keys",
+    CASE(CheckCubRedundantFunctionCall(),
```
Comment (zhimingwang36) [inline clang/lib/DPCT/APINamesCUB.inc]: please don't re-format inc in this PR, keep footprint as small as possible.

Comment (zhiweij1) [reply clang/lib/DPCT/APINamesCUB.inc]: done

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**